contracts/token/SRC20.sol

```diff
@@ -354,6 +338,8 @@ contract SRC20 is ISRC20, ISRC20Owned, ISRC20Managed, SRC20Detailed, Ownable {
     }
 
     function transfer(address to, uint256 value) external returns (bool) {
+        require(!_features.checkTransfer(msg.sender, to));
```

Comment:
`checkTransfer(from, to)` has this condition:
```
return _isAccountFrozen(from) && _isAccountFrozen(to) && paused();
```
If `from` is not frozen, all other combinations will result in allowed transfer. Same applies everywhere `checkTransfer` is used.

Reply:
Well spotted, thanks. Fix applied.